README.md

```diff
@@ -11,153 +11,19 @@ Official GitHub Action for running load tests with <a href="https://artillery.io
 
 The `artilleryio/action-run` action supports a subset of the [`run` command options](https://www.artillery.io/docs/reference/cli/run#options) of our CLI.
 
-### `test`
+### `command`
 
-A test script path.
+A path to the test script to run.
```

Comment:
This should be updated no?

e.g. `The Artillery CLI command to run.`

Reply:
Done!